softgroup/basis/common-protocol/src/main/java/com/softgroup/common/protocol/Status.java

```diff
@@ -0,0 +1,22 @@
+package com.softgroup.common.protocol;
+
+/**
+ * Created by polev on 05.04.2017.
+ */
+public enum Status {
```

Comment:
@MariiaPolova Status of what? Next - each value must have integer code and corresponding string description